### DISCUSSION TITLE
A question about Unity build IL2CPP Code Generation

### DISCUSSION THREAD

**Discussion by a13782425** [timcassell/ProtoPromise#452] [Q&A] [answered]
When I set IL2CPP Code Generation to Faster runtime, the Android project was abnormally large, which sometimes resulted in the failure to export the package.

**Comment by timcassell** [comment, marked as answer]
How are you building? I recently ran into a disk space issue on my CI. See https://github.com/game-ci/unity-builder/issues/656 for more info. It seems Android builds tend to be larger than other targets, and there's nothing I can do about it. You will either need to increase your disk space, or use IL2CPP smaller build option.

**Reply by a13782425** [reply]
It's the same problem, but there doesn't seem to be a particularly good solution at the moment except to increase disk capacity
